ModelBundle/Tests/Functional/Repository/ContentRepositoryTest.php

```diff
@@ -93,13 +93,18 @@ public function provideFindOneByContentId()
     }
 
     /**
-     * @param string $contentId
+     * @param $contentId
+     * @param $version
+     * @param string|null $language
      *
      * @dataProvider providefindLastPublishedVersionByContentIdAndLanguage
      */
     public function testFindLastPublishedVersionByContentIdAndLanguage($contentId, $version, $language = null)
     {
-        $content = $this->repository->findOneByContentId($contentId);
+        if (null === $language) {
```

Comment:
instead, you should modify the dataProvider, the language cannot be null anymore